docs/pages/repo/docs/reference/configuration.mdx

```diff
@@ -8,13 +8,13 @@ import OutputModeTable from "../../../../components/output-mode-table.mdx";
 
 # Configuration Options (`turbo.json`)
```

Comment:
Tangential to the PR, but does it not make sense for this to be generated from our schema.json at some point? Not only would it avoid drift but we could move these docs into the schema so they are available in the IDE as well.

Reply:
That makes sense to me